multi-repo/argo-app-of-apps/dev/templates/ecsdemo-crystal.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
It is a bit weird for the customers to see  "ecs" all over the files when these are clearly EKS demoes. I realize it is not something that we invented. so it is just an observation. Ideally should be fixed in the respective repo.

Reply:
Agreed. 
This can be taken as a next step to create respective repositories without any orchestrator reference in the repository name